app/models/product.rb

```diff
@@ -0,0 +1,7 @@
+class Product < ApplicationRecord
+
+    def pretty_info
+        "#{title} at #{price}"
```

Comment:
Hmmmm do you think my pretty_info method should have more stuff?

Reply:
Yes, I do